packages/imperative/src/rest/src/session/ConnectionPropsForSessCfg.ts

```diff
@@ -358,7 +380,7 @@ export class ConnectionPropsForSessCfg {
      * @param connOpts Options for adding connection properties
      * @returns Name-value pairs of connection properties
      */
-    private static getValuesBack(connOpts: IOptionsForAddConnProps): (properties: string[]) => Promise<{ [key: string]: any }> {
+    private static getValuesBack<SessCfgType extends ISession=ISession>(connOpts: IOptionsForAddConnProps<SessCfgType>): (properties: string[]) => Promise<{ [key: string]: any }> {
```

Comment:
Sorry for being picky, and this is not really a required change, but this line gets flagged by the linter as a warning because it's longer than 150 characters 😋